heareval/score.py

```diff
@@ -171,19 +191,34 @@ class EventBasedScore(SoundEventScore):
     event-based scores - the ground truth and system output are compared at
     event instance level;
 
-    See https://tut-arg.github.io/sed_eval/generated/sed_eval.sound_event.EventBasedScores.html # noqa: E501
+    See https://tut-arg.github.io/sed_eval/generated/sed_eval.sound_event.EventBasedMetrics.html # noqa: E501
     for params.
     """
 
     score_class = sed_eval.sound_event.EventBasedMetrics
 
 
 available_scores: Dict[str, Callable] = {
-    "top1_err": Top1Error,
-    "pitch_err": partial(Top1Error, name="pitch_err"),
-    "chroma_err": ChromaError,
-    "onset_only_event_based": partial(
-        EventBasedScore, params={"evaluate_offset": False, "t_collar": 0.2}
+    "top1_acc": Top1Accuracy,
+    "pitch_acc": partial(Top1Accuracy, name="pitch_acc"),
+    "chroma_acc": ChromaAccuracy,
```

Comment:
The event glomming hyperparams belong here too somehow.

The score names are getting pretty long but at least they're specific

Reply:
Ya cool -- this is sort of how I imagined this would go, I had thought about moving these score params into the task config dict where we specify the eval metric -- but I think this is okay here.